pkg/apis/backups/v1alpha1/migrate_type.go

```diff
@@ -90,20 +89,50 @@ type MigratePolicy struct {
 	PreserveNodePorts *bool `json:"preserveNodePorts,omitempty"`
 }
 
+// MigratePhase is a string representation of the lifecycle phase of a Migrate instance
+// +kubebuilder:validation:Enum=New;FailedValidation;WaitingForSource;InProgress;Completed;Failed
+type MigratePhase string
+
+const (
+	// MigratePhaseNew means the migrate has been created but not
+	// yet processed by the RestoreController
+	MigratePhaseNew MigratePhase = "New"
+
+	// MigratePhaseFailedValidation means the migrate has failed
+	// the controller's validations and therefore will not run.
+	MigratePhaseFailedValidation MigratePhase = "FailedValidation"
+
+	// MigratePhaseWaitingForSource means the migrate is currently fetching source cluster resource.
+	MigratePhaseWaitingForSource MigratePhase = "WaitingForSource"
+
+	// MigratePhaseSourceReady means the migrate is already currently fetched source cluster resource.
+	MigratePhaseSourceReady MigratePhase = "SourceReady"
```

Comment:
can we see this phase in eye, i am not against this phase, just a doubt

Reply:
you are right, this phase may be unnesscery.